src/database/migrations/20250226111922-request_session_requests_table.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('request_session_requests', {
```

Comment:
@sumanvpacewisdom what is the difference b.w this one src/database/migrations/20250211135450-request_sessions_table.js